docs/tutorials/Manual-Signature-Verification/Readme.md

```diff
@@ -0,0 +1,268 @@
+| `signatureVersion` | Signature version. | Comes in the signature HTTP header. |
+
+You will receive the following response:
+
+```json
+{
+  "status": "string",
+  "responseObject": {
+    "signatureValid": true,
+    "activationId": "string",
+    "activationStatus": "ENUM_ACTIVATION_STATUS",
+    "userId": "string",
+    "applicationId": 0,
+    "blockedReason": "string",
+    "remainingAttempts": 0,
+    "signatureType": "ENUM_SIGNATURE_TYPE"
+  }
+}
+```
+
+**Response Body Structure**
+
+| Parameter | Description |
+|---|---|
+| `status` | Status of the request processing, `OK` if the HTTP request executes correctly, `ERROR` if an error occurs. |
+| `responseObject.signatureValid` | Boolean indicating if signature was verified correctly or if the signature was invalid. |
+| `responseObject.activationId` | Activation ID of the activation used to verify the signature. |
+| `responseObject.activationStatus` | Status of the activation during the verification attempt (`ACTIVE`, `BLOCKED`, `REMOVED`, unless you made an implementation error, you should not obtain `CREATED` or `PENDING_COMMIT` values.) |
+| `responseObject.userId` | User ID of the user who is associated with this activation. |
+| `responseObject.applicationId` | Application ID of an application that was used to compute the signature. |
+| `responseObject.blockedReason` | In case the activation is blocked, this value provides more information about the cause. It can be any value provided while calling the service that blocks the activation, or a `MAX_FAILED_ATTEMPTS` value in case the activation was blocked due to too many authentication failed attempts. |
+| `responseObject.remainingAttempts` | Number of the remaining authentication attempts. |
+| `responseObject.signatureType` | Used signature type. |
+
+_Note: We also have a SOAP service. If you like this better, [here is the documentation](https://developers.wultra.com/docs/2020.05/powerauth-server/SOAP-Service-Methods), including the link to WSDL definitions._
+
+The first thing you must do is to check the `responseObject.signatureValid` value. If the value is `false`, the signature verification failed and you should return `HTTP 401` in the response to the mobile client.
+
+If the signature verification was successful, you can now work with the remaining response values in any way your business logic requires.
+
+For example, you can:
+
+- Obtain the user ID to create authenticated session or issue an access token in case of a login endpoint, or submit a payment while checking the user is allowed to do so in the case of payment endpoint.
+- Check if the used signature type can be used for the given operation (for example, to disable biometry for payments above some value).
+- Check that the application is allowed to perform the operation (for example, your mobile banking app may be able to call all endpoints, while a mobile brokerage app could be restricted just to some endpoints).
+
+## Summary
```

Comment:
It may be useful to mention that:
- there is a PA server client available
- powerauth-cmd-tool can be used for testing signatures